src/configure/configure.ts

```diff
@@ -432,16 +432,18 @@ class PipelineConfigurer {
 
     private async getAzureResourceDetails(): Promise<void> {
         // show available subscriptions and get the chosen one
-        let subscriptionList = extensionVariables.azureAccountExtensionApi.filters.map((subscriptionObject) => {
+        let subscriptionList = extensionVariables.azureAccountExtensionApi.subscriptions.map((subscriptionObject) => {
```

Comment:
should this remain filters ? or do we want to show all accessible subscriptions independent of Azure-account extension settings ?

Reply:
I will stick with filters for now. User has decided to now work with an extension and we should honor that.